aaf_walking_group/www/song.html

```diff
@@ -81,6 +87,20 @@
 			var result = aafMusicPlayerService(1);
 			
 		}
+		
+		function quieter() {
+      console.log('Quieter Called');
+		var service_button = new ROSLIB.Service({ros : ros, name : '/walking_group/guide_interface/volume_quieter', serviceType : 'strands_webserver/CallButton'}); 
```

Comment:
https://github.com/strands-project/aaf_deployment/blob/6ff8d3f86c925b944fd31a7088e1e1c54bfba4af/aaf_walking_group/src/aaf_walking_group/guide_interface.py#L30-L31

It's a bit confusing that it is there but this is where the buttons were used before. Nevertheless, the service exists during the whole duration of the state machine running.